libpicotcp/CMakeLists.txt

```diff
@@ -72,6 +72,11 @@ if(LibPicotcp)
     add_custom_target(get_muslc)
     add_dependencies(get_muslc muslc)
 
+    set(sed_inplace_suffix "")
+    if(CMAKE_HOST_APPLE)
+        set(sed_inplace_suffix "''")
+    endif()
```

Comment:
How does GNU `sed` treat `-i ''`? Would that happen do work on both BSD and GNU?

Reply:
Unfortunately not, `-i''` does seem to work though.

Reply:
Ok. Might be worth mentioning in the comment, because I can see us trying that again in the future.